aerogear/data-sync-ui
# Subscriptions dropdown

## Motivation
https://issues.jboss.org/browse/AEROGEAR-7760

## What
Adds a dropdown to add a subscription to a Resolver.

## Why
Because the JIRA says so.

## How
Subscriptions are fetched inside the component, right now the query does not work. (probably need some changes to DB related logic)

## Verification Steps

1. Go to Resolvers and select a resolver.
2. Verify there is a dropdown for subscriptions.
    - If there is no subscriptions, a message is shown instead of a dropdown.
    - If there are subscriptions but none is selected, show form empty.

## Checklist:


## Progress

- [x] Finished UI
- [x] Fix DB related logic (getSubscriptions not working)
- [x] Add tests for the UI
- [x] Add some test for getSubscriptions
